lib/db.mysql.js

```diff
@@ -51,7 +51,7 @@ if (api.mysql) {
     });
 
     connection.on('error', err => {
-      database.application.log.error(err);
+      database.application.log.error(err.toString());
```

Comment:
```suggestion
      database.application.log.error(err.stack);
```
Such change should be performed everywhere across all files.